### PR TITLE
{tools}[GCCcore/12.2.0] archspec v0.2.0 w/ Python 3.10.8

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bwd-rockylinux-91 - Linux Rocky Linux 9.1 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), 2 x NVIDIA NVIDIA GeForce GTX 1060 6GB, 515.86.01, Python 3.9.14
See https://gist.github.com/f264dd66908cb8d420662fd928ddd8dd for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/a/archspec/archspec-0.2.0-GCCcore-12.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+dependencies = [('Python', '3.10.8')]
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
+sanity_check_commands = ["python -c 'from archspec.cpu import host; print(host())'"]
+
+moduleclass = 'tools'
+
```
Comment (jfgrimm) [inline easybuild/easyconfigs/a/archspec/archspec-0.2.0-GCCcore-12.2.0.eb:25]:
```suggestion
```

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**FAILED**
Build succeeded for 33 out of 38 (1 easyconfigs in total)
himem05.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6230 CPU @ 2.10GHz (cascadelake), Python 3.6.8
See https://gist.github.com/6c1d3694ee974ecbeddac4646665a507 for a full test report.

**Comment by SebastianAchilles** [comment]
@jfgrimm This error in `libxml2` looks familiar. Are you using the updated `libxml2` easyblock from https://github.com/easybuilders/easybuild-easyblocks/pull/2825? This update is also part of the EasyBuild v4.7.0 https://github.com/easybuilders/easybuild-easyconfigs/pull/17065.

**Review by jfgrimm — APPROVED** [review]
lgtm

**Comment by jfgrimm** [comment]
Going in, thanks @SebastianAchilles!

**Pull request merged**